emcc.py

```diff
@@ -2767,6 +2771,7 @@ def get_eliminate():
 def parse_args(newargs):
   options = EmccOptions()
   settings_changes = []
+  user_settings = []
```

Comment:
`user_settings` doesn't feel right to me, somehow. how about `user_js_defines`? that matches more closely with `jsD / js define`.

Reply:
Sounds good.